src/container.cpp

```diff
@@ -446,63 +444,63 @@ Cylinder* Container::queryDestination(int32_t& index, const Thing& thing, Item**
 		the slot position as if the bag has 20 slots
 		*/
 		index = INDEX_WHEREEVER;
-		*destItem = nullptr;
+		destItem = nullptr;
 	}
 
-	const Item* item = thing.getItem();
+	auto item = thing->getItem();
```

Comment:
Could this be `const auto`
```suggestion
	const auto item = thing->getItem();
```